### PR TITLE
remove NSX references from Cloud 8 Crowbar deployment (SOC-10081)

### PR DESCRIPTION
**Pull request by csymons-suse**
NSX on SOC 8 Crowbar was not tested or qualified.

### CODE REVIEW

File: xml/depl_nodes.xml [hunk 1]
```diff
@@ -3067,22 +3067,22 @@ host1:/srv/nfs/share1 /mnt/nfs/share1 -o rsize=8192,wsize=8192,timeo=14,intr
   </bridgehead>
 
   <para>
-   This plug-in requires to configure access to the &vmware; NSX service.
+   This plug-in requires configuring access to the &vmware; service.
```
Comment (asettle) [inline xml/depl_nodes.xml]: `This plug-in is required to configure access to the &vmware; service.` ?

Feels like the initial variation was missing bits

File: xml/depl_nodes.xml [hunk 2]
```diff
@@ -2861,11 +2861,11 @@ host1:/srv/nfs/share1 /mnt/nfs/share1 -o rsize=8192,wsize=8192,timeo=14,intr
   </para>
 
   <para>
-   The <guimenu>vmware</guimenu> option lets you use an existing &vmware; NSX
-   installation. Using this plugin is not a prerequisite for the &vmware; vSphere
-   hypervisor support. However, it is needed when wanting to have security
-   groups supported on &vmware; compute nodes. For all other scenarios, choose
-   <guimenu>ml2</guimenu>.
+   The <guimenu>vmware</guimenu> option lets you use an existing &vmware;
+   installation. Using this plugin is not a prerequisite for the &vmware;
```
Comment (asettle) [inline xml/depl_nodes.xml]: Just a note, below you've got "plug-in" and here it's "plugin" :)

File: xml/depl_nodes.xml (hunk 1 above)
Comment (asettle) [reply xml/depl_nodes.xml]: This is the only change that still doesn't make sense to me. @dmpop second eyes, please? :)

Comment (csymons-suse) [reply xml/depl_nodes.xml]: give this version a whirl. Thanks for sticking with it. No response from tech folks.

File: xml/depl_vmware.xml [hunk 3]
```diff
@@ -67,9 +67,8 @@
    </listitem>
    <listitem>
     <para>
-     Security groups are only supported when running &vmware; NSX. You need to
-     deploy &o_netw; with the <guimenu>vmware</guimenu> plug-in to have
-     security group support. This is also a prerequisite for
+     You need to deploy &o_netw; with the <literal>vmware</literal> plug-in to
```
Comment (tbosworth7) [inline xml/depl_vmware.xml]: I believe this sentence should be removed.

File: xml/depl_nodes.xml [hunk 4]
```diff
@@ -3067,22 +3067,22 @@ host1:/srv/nfs/share1 /mnt/nfs/share1 -o rsize=8192,wsize=8192,timeo=14,intr
   </bridgehead>
 
   <para>
-   This plug-in requires to configure access to the &vmware; NSX service.
+   This plug-in is required for configuring access to the &vmware; service.
```
Comment (tbosworth7) [inline xml/depl_nodes.xml]: This whole section is only for NSX so it should be taken out.  I noticed that this stuff crept into the Cloud 7 docs as well not sure how that happened.

File: xml/depl_nodes.xml [hunk 5]
```diff
@@ -3067,22 +3067,22 @@ host1:/srv/nfs/share1 /mnt/nfs/share1 -o rsize=8192,wsize=8192,timeo=14,intr
...
-      Login credentials for the &vmware; NSX server. The user needs to have
-      administrator permissions on the NSX server.
+      Login credentials for the &vmware; server. The user needs to have
+      administrator permissions on the server.
      </para>
     </listitem>
    </varlistentry>
    <varlistentry>
-    <term><guimenu>&vmware; NSX Controllers</guimenu>
+    <term><guimenu>&vmware; Controllers</guimenu>
```
Comment (tbosworth7) [inline xml/depl_nodes.xml]: This section is only for NSX so it should be removed.

File: xml/depl_nodes.xml [hunk 6]
```diff
@@ -2861,10 +2861,10 @@ host1:/srv/nfs/share1 /mnt/nfs/share1 -o rsize=8192,wsize=8192,timeo=14,intr
   </para>
 
   <para>
-   The <guimenu>vmware</guimenu> option lets you use an existing &vmware; NSX
-   installation. Using this plugin is not a prerequisite for the &vmware; vSphere
-   hypervisor support. However, it is needed when wanting to have security
-   groups supported on &vmware; compute nodes. For all other scenarios, choose
+   The <guimenu>vmware</guimenu> option lets you use an existing &vmware;
+   installation. Using this plug-in is not a prerequisite for the &vmware;
+   vSphere hypervisor support. However, it is needed to have security groups
```
Comment (tbosworth7) [inline xml/depl_nodes.xml]: The last sentence should be removed. "However..."

**Pull request merged**